build/.map

```diff
@@ -0,0 +1,79 @@
+# Scripts Map (Scripts Call Tree, shows the relations between the scripts).
+# Azure Pipelines documentation: https://docs.microsoft.com/en-us/azure/devops/pipelines/?view=azure-devops
+# See also: 
+#   QDK build pipeline wiki https://ms-quantum.visualstudio.com/Quantum%20Program/_wiki/wikis/Quantum-OKRs.wiki/328/QDK-build-pipeline
+
+# Pipeline: 
+#   microsoft.qsharp-runtime            https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build?definitionId=15&_a=summary
+#   microsoft.qsharp-runtime.sanitized  https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build?definitionId=34&_a=summary
+ci.yml (Build.Configuration = (microsoft.qsharp-runtime           ? Release : 
+                               microsoft.qsharp-runtime.sanitized ? Debug   :
+                               <?>), 
+        Build.Verbosity = n, Skip.Tests = false)
+    steps.yml
+        steps-init.yml
+            'Use NuGet 5.6.0' NuGetToolInstaller@0
```

Comment:
is this file generated or manually written? If generated, then I would not expect it to be checked in. If manually written, it's going to be a pain point keeping it up to date. Do we really need to check this into the repo? It seems like it is all duplicate information.

Reply:
This is manually written. I have written this for myself when understanding how the pipelines work, and I decided to share this with the team (and with myself in the future).

I was assuming that this file does not need to be kept up to date. But when one needs to learn the pipelines again, they will have some starting point and update the file upon need.

Also if we have a problem in the pipeline, then looking at this map file we can quickly get an idea in which .yml or .ps1 to look for a problem.

Reply:
I think putting something like this on the wiki might make sense, but having this in repo feels like it could get confusing. I'd personally prefer to not check it in here.

Reply:
OK, Removing.